src/sentry/static/sentry/app/views/organizationIntegrations/IntegrationDirectorySentryAppRow.tsx

```diff
@@ -148,15 +128,13 @@ const color = {
   [PENDING]: 'yellowOrange',
 };
 
-type StatusIndicatorProps = {status: string; theme?: any; isInternal: boolean};
+type StatusIndicatorProps = {status: string; theme?: any};
 
 const StatusIndicator = styled(({status, ...props}: StatusIndicatorProps) => {
-  //need to omit isInternal
-  const propsToPass = omit(props, ['isInternal']);
   return (
     <FlexContainer>
       <CircleIndicator size={6} color={theme[color[status]]} />
-      <div {...propsToPass}>{t(`${status}`)}</div>
+      <div {...props}>{t(`${status}`)}</div>
```

Comment:
@salty-horse do we really need `{...props}` here?

Reply:
@scefali we need it. WIthout it, the div loses some styling.